{{ cookiecutter.package_name }}/docs/index.rst

```diff
@@ -1,14 +1,17 @@
-Documentation
-=============
+{{ '*' * "%s Documentation"|format(cookiecutter.package_name)|length() }}
+{{ cookiecutter.package_name }} Documentation
+{{ '*' * "%s Documentation"|format(cookiecutter.package_name)|length() }}
 
 This is the documentation for {{ cookiecutter.package_name }}.
 
 .. toctree::
   :maxdepth: 2
 
-  {{ cookiecutter.module_name }}/index.rst
+  subpage.rst
```

Comment:
I think I would call this page ``example_subpkg`` to match the sub-package in the package, and also would only include the toctree and example_subpkg if ``include_example_code`` is ``y``.

Reply:
I did it but that also means that if people don't want example code, they are going to get the subpage as orphan. Is this desirable?

Reply:
I think this is the only thing holding up the PR?